src/Microsoft.Azure.WebJobs.ServiceBus/EventHubs/EventHubListener.cs

```diff
@@ -116,7 +122,8 @@ public Task OpenAsync(PartitionContext context)
 
             public Task ProcessErrorAsync(PartitionContext context, Exception error)
```

Comment:
Do you have any way to verify this is getting called and logged? I wonder if there's an e2e test we can write that exercises this handling somehow.

Reply:
I could not find figure out a way to test this.  Let me know if there is anyone we could contact to get a working sample that triggers an error.

Reply:
I have opened issue #1604 for investigating end to end testing. Will go ahead and merge this PR